waku/v2/node/wakunode2.go

```diff
@@ -92,7 +93,7 @@ type WakuNode struct {
 
 	relay           Service
 	lightPush       Service
-	discoveryV5     Service
+	discoveryV5     *discv5.DiscoveryV5
```

Comment:
What's the advantage of using the specific type here compared to using the `Service` interface?